src/wallet/wallet2.cpp

```diff
@@ -5688,6 +5720,51 @@ void wallet2::get_outs(std::vector<std::vector<tools::wallet2::get_outs_entry>>
     THROW_WALLET_EXCEPTION_IF(resp_t.result.status == CORE_RPC_STATUS_BUSY, error::daemon_busy, "get_output_histogram");
     THROW_WALLET_EXCEPTION_IF(resp_t.result.status != CORE_RPC_STATUS_OK, error::get_histogram_error, resp_t.result.status);
 
+    // if we want to segregate fake outs pre or post fork, get distribution
+    std::unordered_map<uint64_t, std::pair<uint64_t, uint64_t>> segregation_limit;
+    float min_pre_fork_ratio = 0.0f, min_post_fork_ratio = 0.0f;
```

Comment:
Looks like these aren't used anywhere?

Reply:
They're used to calculate pre/post _fork_outputs_count below.

Reply:
Really? Which line? I’m asking this because I couldn’t find after word search.

Reply:
Really? Which line? I’m asking this because I couldn’t find after word search.

Reply:
line 5949, in the "how many fake outs to draw on a pre-fork triangular distribution" part.

Reply:
Those are `pre_fork_num_out_ratio` and `post_fork_num_out_ratio`; here these are `min_pre_fork_ratio` and `min_post_fork_ratio`.

Reply:
Ah, I'd missed them, removed.